src/context.rs

```diff
@@ -751,6 +760,19 @@ impl Context {
         Ok(list)
     }
 
+    /// Returns oldest fresh message in unmuted and unblocked chats.
+    ///
+    /// If there are no such messages, waits until there is one.
```

Comment:
> waits until there is one.

is this also true for cffi/jsonprc? if so, this information should be added to the docs there as well.

maybe also add a line for the usecase. though i am sure, there is concrete need somewhere, the uscase is not clear to at least to me :)

Reply:
hm, get_next_event() is called on an explicit event emitter object that does not have many other usages, so that seems less confusing than for the context where all the other get() work differently.

but i agree, wait() would have been better for the context emitter as well.

i still suggest to not repeat the "mistake" use sth. with "wait" in the name here. maybe we can rename the other usages accordingly when we iterate over or so.